client/state/reader/streams/schema.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+export const itemsSchema = {
+	type: 'object',
+	patternProperties: {
+		'(w|d)+': {
```

Comment:
is that a valid pattern?

Reply:
it worked in my (too limited) test cases! 
After checking on https://www.jsonschemavalidator.net/ I think a better pattern would be:
`\\w|\\d`. This would match anything containing either letters or numbers.